tests/test_fixtures_scenarios.py

```
@@ -0,0 +1,83 @@
+            "my_float": 3.14,
+            "my_string": "test",
+            "my_enum": "opt1"
+        }
+    }
+
+    # Test updating values
+    app_state.set_data_by_path("root/config.yaml", "my_int", 100)
+    assert app_state.get_data_by_path("root/config.yaml/my_int") == 100
+
+    app_state.set_data_by_path("root/config.yaml", "my_bool", False)
+    assert app_state.get_data_by_path("root/config.yaml/my_bool") == False
```

Ruff (0.15.20)

[error] Avoid equality comparisons to `False`; use `not ...:` for false checks

Replace comparison

(E712)